arm-web/2016-09-01/swagger/AppServiceEnvironments.json

```diff
@@ -1861,6 +1861,250 @@
     },
     "AppServiceEnvironment": {
```

Comment:
Yes this is intentional. Same explanation as AppServiceCertificate vs AppServiceCertificateResource. Both models are used.